hive/replays/circular_replay.py

```diff
@@ -242,7 +242,7 @@ def sample(self, batch_size):
 
         if self._n_step == 1:
             is_terminal = terminals
-            trajectory_lengths = np.ones(batch_size)
+            trajectory_lengths = np.ones(terminals.shape[0])
```

Comment:
for the ppo agent, in case of stack_size > 1, if we have a episode reset in between the replay, then the new padded transitions are not valid indices yet they contribute to the size. So it leads to the last batch with transitions less than batch_size, and this part throws error.

Reply:
Ok, I think if you change the api as i mention in ppo.py, you'll need to override sample anyway, in which case this change wouldn't be necessary. Let me know if you have any comments on the proposed changes.